frontend/src/components/home/About.js

```diff
@@ -1,7 +1,25 @@
+import { Button, Typography } from '@mui/material'
 import React from 'react'
+import { Box } from '@mui/material'
 
 function About() {
-    return <div>-about component placeholder for home page-</div>
+    return <div>
+        <Box sx={{ textAlign: "right", padding: "100px" }}>
```

Comment:
Good work, we just need to lose the fontFamily and use the default to match with the other component designs and also figma. Otherwise everything looks good!